src/lib/updating-element.ts

```diff
@@ -412,11 +453,14 @@ export abstract class UpdatingElement extends HTMLElement {
       return;
     }
     const type = options.type;
-    const converter = options.converter;
-    const toAttribute =
-        converter && (converter as ComplexAttributeConverter).toAttribute ||
-        defaultConverter.toAttribute;
-    return toAttribute!(value, type);
+    let converter = options.converter;
+    let toAttribute =
+        converter && (converter as ComplexAttributeConverter).toAttribute;
+    if (!toAttribute) {
+      converter = defaultPropertyDeclaration.converter;
+      toAttribute = converter && (converter as ComplexAttributeConverter).toAttribute;
+    }
+    return toAttribute ? toAttribute(value, type) : value;
```

Comment:
Didn't follow the significance of the change from `defaultConverter` to `defaultPropertyDeclaration.converter`, nor how `toAttribute` could end up being falsey by the end here....

Reply:
From another refactoring. Will revert...